__tests__/app.test.js

```diff
@@ -20,4 +25,14 @@ describe.only("GET /api", () => {
         console.log(endpointsJson)
       });
   });
+  test("200: Responds with an array of object with slug and description keys",()=>{
+    return request (app)
+    .get("/api/topics")
+    .expect(200)
+    .then(({body: {topics}})=>{
+      console.log(topics,"<---- this is topic")
+      expect(topics).toBeInstanceOf(Array)
+      expect(topics.length).toEqual(3)
```

Comment:
I like that you have checked the length of this array, however, we also want to make sure that our topics have the keys we'd expect them to have. In this case, it would be a Slug and a Description. Maybe refactor this to using a forEach and checking that they all have a Slug + Description (which should be strings!)